database/pgSeed.js

```diff
@@ -0,0 +1,102 @@
+// seed 10M rows
+const faker = require('faker');
+const fs = require('file-system');
+const _progress = require('cli-progress');
+
+const randomLorem = faker.lorem.word;
+const productType = [
+  'Chair',
+  'Dresser',
+  'Drawer',
+  'Desk',
+  'Table',
+  'Island',
+  'Rug',
+  'Towel',
+  'Lamp',
+  'Box',
+  'Storage',
+  'Mirror',
+  'Stool',
+  'Bench',
+  'Curtain',
+  'Organizer',
+  'Bowl',
+  'Cutlery'
+];
+const dimensions = [
+  'null',
```

Comment:
I'm curious, why is there a null dimension? Not a critique, genuinely don't know.

Reply:
i saw that some products on the ikea website don't show their dimensions in their subheading (at least in the search bar) - so i included a null so some of them won't display that detail